### PR TITLE
remove hosted powergate docs

### PR DESCRIPTION
**Pull request by longfeiWan9**
In the `docs -> build` section, there is two docs are confusing, they are [hosted powergate](https://docs.filecoin.io/build/hosted-powergate/) and [powergate](https://docs.filecoin.io/build/powergate/). After I learn about them, I found out that hosted powergate is supposed to be endpoint service which is similar to Gliff Node, but it is not supported by Textile team anymore. so I will recommend to remove it and add useful content to [powergate](https://docs.filecoin.io/build/powergate/) page.

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/build/powergate.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://godoc.org/github.com/textileio/powergate/api/client
- This phrase is redundant. Consider writing “facilitate”.
  ```
  ...s higher-level APIs for developers that facilitate easier interaction with I...
                                             ^^^^^^^^^^^^^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “their”.
  ```
  ...from Filecoin, but who prefer to manage their own nodes, and also gain acces...
                                             ^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...les: Replication factor Miner selection Deal renewal and Repair # How to use...
                                             ^^^^
  ```
- ‘to the full’ might be wordy. Consider a shorter alternative.
  ```
  ...with the localnet as you'll have access to the full set of APIs and capabili...
                                             ^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...n, you need to fully sync from Filecoin MainNet which may take over a day. T...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...ou can also choose to sync from current snapshot which was provided by Proto...
                                             ^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...D and a User auth token. Write down the token as we will need it for every o...
                                             ^^^^^
  ```
- Possibly missing comma found.
  ```
  ...the IPFS network, you don't need to run stage as the Powergate will automati...
                                             ^^^^^
  ```
- Do not use a noun immediately after the pronoun ‘you’. Use a verb or an adverb, or possibly some other part of speech.
  ```
  ...eve your file To retrieve the file that you storage through POW CLI, you can...
                                             ^^^^^^^^^^^
  ```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build failed...

```

> filecoin-docs@1.0.0 docs:build /home/runner/work/filecoin-docs/filecoin-docs
> npm run docs:links && vuepress build docs


> filecoin-docs@1.0.0 docs:links /home/runner/work/filecoin-docs/filecoin-docs
> vuepress check-md docs


Checking markdown...

2 dead links was found

  File is not found: [Build with Powergate](hosted-powergate.md) (/home/runner/work/filecoin-docs/filecoin-docs/docs/build/README.md:20:3)
  File is not found: [hosted](../build/hosted-powergate.md) (/home/runner/work/filecoin-docs/filecoin-docs/docs/store/README.md:47:224)

Executes with --fix to fix automatically

Checking failed

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! filecoin-docs@1.0.0 docs:links: `vuepress check-md docs`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the filecoin-docs@1.0.0 docs:links script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-11-01T10_18_32_271Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! filecoin-docs@1.0.0 docs:build: `npm run docs:links && vuepress build docs`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the filecoin-docs@1.0.0 docs:build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-11-01T10_18_32_297Z-debug.log
```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!


### `docs/build/powergate.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://godoc.org/github.com/textileio/powergate/api/client
- This phrase is redundant. Consider writing “facilitate”.
  ```
  ...s higher-level APIs for developers that facilitate easier interaction with I...
                                             ^^^^^^^^^^^^^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “their”.
  ```
  ...from Filecoin, but who prefer to manage their own nodes, and also gain acces...
                                             ^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...les: Replication factor Miner selection Deal renewal and Repair # How to use...
                                             ^^^^
  ```
- ‘to the full’ might be wordy. Consider a shorter alternative.
  ```
  ...with the localnet as you'll have access to the full set of APIs and capabili...
                                             ^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...n, you need to fully sync from Filecoin MainNet which may take over a day. T...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...ou can also choose to sync from current snapshot which was provided by Proto...
                                             ^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...D and a User auth token. Write down the token as we will need it for every o...
                                             ^^^^^
  ```
- Possibly missing comma found.
  ```
  ...the IPFS network, you don't need to run stage as the Powergate will automati...
                                             ^^^^^
  ```
- Do not use a noun immediately after the pronoun ‘you’. Use a verb or an adverb, or possibly some other part of speech.
  ```
  ...eve your file To retrieve the file that you storage through POW CLI, you can...
                                             ^^^^^^^^^^^
  ```

File: docs/build/powergate.md [hunk 1]
```diff
@@ -29,15 +27,102 @@ There are many ways to interact with the Powergate. These pathways are well-docu
...
+   ```
+   pow data get <cid> fileName
+   ```
+
+1. **Interact with the Powergate API**:
+
+   Once you are familiar with the Powergate workflows, you can start programatically controlling the Powergate instance. Textile provides Go and Javascript API clients for Powergate using the gRPC API endpoint:
+
+   - [js-powergate-client](https://github.com/textileio/js-powergate-client)
+   - [Go Powergate client module](https://godoc.org/github.com/textileio/powergate/api/client)
```
Comment (johnnymatthews) [inline docs/build/powergate.md:118]:
```suggestion
   - [Go Powergate client module](https://pkg.go.dev/github.com/textileio/powergate/api/client)
```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!


### `docs/build/powergate.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://godoc.org/github.com/textileio/powergate/api/client
- This phrase is redundant. Consider writing “facilitate”.
  ```
  ...s higher-level APIs for developers that facilitate easier interaction with I...
                                             ^^^^^^^^^^^^^^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “their”.
  ```
  ...from Filecoin, but who prefer to manage their own nodes, and also gain acces...
                                             ^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...les: Replication factor Miner selection Deal renewal and Repair # How to use...
                                             ^^^^
  ```
- ‘to the full’ might be wordy. Consider a shorter alternative.
  ```
  ...with the localnet as you'll have access to the full set of APIs and capabili...
                                             ^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...n, you need to fully sync from Filecoin MainNet which may take over a day. T...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...ou can also choose to sync from current snapshot which was provided by Proto...
                                             ^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...D and a User auth token. Write down the token as we will need it for every o...
                                             ^^^^^
  ```
- Possibly missing comma found.
  ```
  ...the IPFS network, you don't need to run stage as the Powergate will automati...
                                             ^^^^^
  ```
- Do not use a noun immediately after the pronoun ‘you’. Use a verb or an adverb, or possibly some other part of speech.
  ```
  ...eve your file To retrieve the file that you storage through POW CLI, you can...
                                             ^^^^^^^^^^^
  ```

**Comment by johnnymatthews** [comment]
This is great, thanks for this PR @longfeiWan9. I've made a few minor edits, mainly the suggestions from the @filecorgi bot.

**Review by johnnymatthews — APPROVED** [review]

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!


### `docs/build/powergate.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://godoc.org/github.com/textileio/powergate/api/client
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...les: Replication factor Miner selection Deal renewal and Repair # How to use...
                                             ^^^^
  ```
- Possibly missing comma found.
  ```
  ...n, you need to fully sync from Filecoin MainNet which may take over a day. T...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...the IPFS network, you don't need to run stage as the Powergate will automati...
                                             ^^^^^
  ```

**Comment by longfeiWan9** [comment]
> This is great, thanks for this PR @longfeiWan9. I've made a few minor edits, mainly the suggestions from the @filecorgi bot.

Great. thank you for adding more commits which is very helpful. I am not authorized to merge PR here. So if you think it is good to go, please help me to merge it.

**Pull request merged**